### PR TITLE
Changes the organization feature implementation

### PR DESCRIPTION
**Pull request by oesukam**
#### What does this PR do?
Changes the organization feature implementation

#### Description of Task to be completed?
- [x] Implement create an organization feature
- [x] Implement retrieve a single organization feature
- [x] Implement update an organization feature
- [x] Write integration test(s)

#### How should this be manually tested?
- Clone the repo
- Run `npm run test`

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A

### CODE REVIEW

File: src/routes/api/v1/organizations.js [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
 router
-    .route('/:organizationId').get(checkOrganization, asyncHandler(OrganizationController.get));
+  .route('/:organizationId')
+  .all(asyncHandler(checkOrganization))
+  .get(asyncHandler(OrganizationController.get))
+  .put(
+    checkAuth,
+    celebrate({ body: organizationValidator.update }),
+    asyncHandler(OrganizationController.update),
+  );
```
Comment (gracelungu) [inline src/routes/api/v1/organizations.js:31]: @oesukam Could we also have a delete feature for organizations?

**Review by gracelungu — APPROVED** [review]

File: src/controllers/PostController.js [hunk 2]
```diff
@@ -134,7 +134,7 @@ export default class PostController {
   static async getPosts(req, res) {
     const posts = await Post.find({})
       .select('-__v')
-      .populate('author', '-_id -__v -userType -password');
+      .populate('author', 'picture username firstName lastName followerCount followedCount country city ');
 
     return res.status(statusCodes.OK).json({
       status: statusCodes.OK,
```
Comment (gracelungu) [inline src/controllers/PostController.js:140]: @oesukam We should also enable the user to post as an organization

**Review by gracelungu — COMMENTED** [review]
Could we refactor feed so that we can be able to only get organization posts?

File: src/controllers/PostController.js (hunk 2 above)
Comment (oesukam) [reply src/controllers/PostController.js:140]: It is still applies. I didn't want to remove the actual user who posted as the organization even if we are not going to display those information @gracelungu

File: src/routes/api/v1/organizations.js (hunk 1 above)
Comment (oesukam) [reply src/routes/api/v1/organizations.js:31]: @gracelungu I can add it

**Comment by gracelungu** [comment]
[Refactor users](https://trello.com/c/Tu1hnJBh/39-refactor-users)

**Pull request merged**